react-front/public/components/DeviceSearchForm.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from 'prop-types'
+import { Button, Select, Input, Icon } from 'semantic-ui-react'
+
+class DeviceSearchForm extends React.Component {
+
+  constructor(props) {
```

Comment:
dependencyn:  "babel-plugin-transform-class-properties" gör så behöver vi inte skriva in constructor(props) {} och super (props) {} utan vi kan starta direkt med state = {} och spara in på lite kod (som i DeviceList.js komponenten).

Reply:
Ok jag har patchat detta!